src/renderer/components/SettingsModal.tsx

```diff
@@ -54,14 +54,22 @@ const AppearanceSettings = memo(() => {
             w="full"
         >
             <DropdownSetting
-                setValue={(value) => setTheme(value as never)}
+                setValue={(value) => setTheme(value)}
                 setting={{
-                    label: 'Select Theme',
-                    description: "Choose the Theme for chaiNNer's appearance.",
+                    label: 'Color Theme',
+                    description: "Choose the Theme for chaiNNer's appearance when in Dark mode.",
                     options: [
-                        { label: 'Dark Mode', value: 'dark' },
-                        { label: 'Light Mode', value: 'light' },
-                        { label: 'System', value: 'system' },
+                        { label: 'Default Dark', value: 'default-dark' },
+                        { label: 'Default Light', value: 'default-light' },
```

Comment:
I think we should keep calling those themes "Light" and "Dark." This should make it easier for users to recognize those themes.

```suggestion
                        { label: 'Dark', value: 'default-dark' },
                        { label: 'Light', value: 'default-light' },
```